packages/closer/pages/subscriptions/summary.tsx

```
@@ -80,8 +80,31 @@ const SubscriptionsSummaryPage: NextPage<Props> = ({
+          });
+        } catch (error) {
+          console.error('Error logging page view:', error);
+        }
+      })();
+      hasComponentRendered.current = true;
+    }
+  }, [selectedPlan]);
+
   useEffect(() => {
-    if (user?.subscription && user.subscription.priceId) {
+    if (user?.subscription && user?.subscription?.priceId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)